lib/wallaroo/core/metrics/reconnecting_metrics_sink.pony

```diff
@@ -852,7 +853,7 @@ actor ReconnectingMetricsSink
         ifdef linux then
           // this is safe because asio thread isn't currently subscribed
           // for a write event so will not be writing to the readable flag
-          AsioEvent.set_writeable(_event, false)
+          AsioEventHelper.set_writeable(_event, false)
```

Comment:
why not @pony_asio_event_set_writeable(_event, false) like `TCPConnection` in mainline Pony?

Reply:
I assumed you had a reason for adding those as methods to `AsioEvent` in our fork of `ponyc`, so I saw this as the minimal change.

Reply:
Oh, I also just assumed you added those, but in fact it was @dipinhora